package.json

```diff
@@ -18,7 +18,7 @@
     "npm": ">=3.0.0"
   },
   "scripts": {
-    "lint": "aegir-lint",
+    "lint": "eslint -c node_modules/aegir/config/eslintrc.yml src test",
```

Comment:
linting is fine, it is just aegir-lint that is being mysterious. This is a known problem by @dignifiedquire, needs to be more investigated. 

thanks @VictorBjelkholm for the tip! :)